src/Console/Report/FixReport/ReporterFactory.php

```diff
@@ -23,11 +24,20 @@
  */
 final class ReporterFactory
 {
+    /** @var array<string, FixerInterface> */
+    private array $fixers;
+
     /**
      * @var array<string, ReporterInterface>
```

Comment:
Please make this one single-line or the others multi-line to have them consistent.